test/protobuf/protoc/proto_gen/extension.pb.ex

```diff
@@ -9,3 +9,27 @@ defmodule Protobuf.Protoc.ExtTest.Foo do
 
   field :a, 1, optional: true, type: :string
 end
+
+defmodule Protobuf.Protoc.ExtTest.UnixDateTime do
+  @moduledoc false
+  use Protobuf, syntax: :proto2
+
+  @type t :: %__MODULE__{
+          microseconds: integer
+        }
+  defstruct [:microseconds]
+
+  field :microseconds, 1, required: true, type: :int64
+end
+
+defmodule Protobuf.Protoc.ExtTest.FooWithDateTime do
+  @moduledoc false
+  use Protobuf, syntax: :proto2
+
+  @type t :: %__MODULE__{
+          inserted_at: DateTime.t() | nil
+        }
+  defstruct [:inserted_at]
+
+  field :inserted_at, 1, optional: true, type: DateTime
```

Comment:
Oops, this is a bug, the `type` here should point to `Protobuf.Protoc.ExtTest.UnixDateTime`

Reply:
Fixed